domain/src/coaching_session.rs

```diff
@@ -85,7 +82,7 @@ pub async fn create(
     }
 }
 
-pub async fn find_by_id(db: &DatabaseConnection, id: entity::Id) -> Result<Option<Model>, Error> {
+pub async fn find_by_id(db: &DatabaseConnection, id: entity::Id) -> Result<Model, Error> {
```

Comment:
Modifying this for ergonomics. (Did a similar thing in https://github.com/refactor-group/refactor-platform-rs/pull/97/commits/80e9996d6593d84a4fe22054825f0c72715a6cb4)